cfn/metrics/publisher.go

```diff
@@ -38,20 +38,22 @@ const (
 
 // A Publisher represents an object that publishes metrics to AWS Cloudwatch.
 type Publisher struct {
-	client    cloudwatchiface.CloudWatchAPI // AWS CloudWatch Service Client
-	namespace string                        // custom resouces's namespace
-	logger    *log.Logger
+	client            cloudwatchiface.CloudWatchAPI // AWS CloudWatch Service Client
+	namespace         string                        // custom resouces's namespace
```

Comment:
it makes more sense to me to have one variable for namespace rather than two for namespace and accountid, as the accountID is a part of the namespace that is later used when putting the metric data

Reply:
Doner